### PR TITLE
config: refactor message queue configuration

### PR DESCRIPTION
**Pull request by dinosk**
* Refactors instantiation of JobStatusConsumer to new message queue
  configuration in reana-commons.

* Removes deprecated configuration.

Connects https://github.com/reanahub/reana/issues/119

Signed-off-by: Dinos Kousidis <dinos.kousidis@cern.ch>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,8 @@ RUN apt-get update && \
 
 COPY CHANGES.rst README.rst setup.py /code/
 COPY reana_workflow_controller/version.py /code/reana_workflow_controller/
+RUN pip install -e git://github.com/dinosk/reana-db.git@119-workflow-submission-queue#egg=reana-db
```
Comment (diegodelemos) [inline Dockerfile]: We can remove since they are released already.

File: reana_workflow_controller/rest.py [hunk 2]
```diff
@@ -1688,6 +1688,89 @@ def _stop_workflow(workflow):
         raise REANAWorkflowControllerError(message)
 
 
+def _run_yadage_workflow_from_spec(workflow):
```
Comment (diegodelemos) [inline reana_workflow_controller/rest.py]: Why do we add these functions? Don't we start workflows only with `_start_workflow` now?

**Comment by diegodelemos** [comment]
Tested together with https://github.com/reanahub/reana-cluster/pull/138, https://github.com/reanahub/reana-client/pull/222 and https://github.com/reanahub/reana-server/pull/116, works like a charm 🚀 just small comments left.

**Comment by dinosk** [comment]
Fixed, thanks for the review

**Pull request merged**